README.md

```diff
@@ -1,17 +1,17 @@
-# SD-Core NRF Operator (k8s)
-[![CharmHub Badge](https://charmhub.io/sdcore-nrf/badge.svg)](https://charmhub.io/sdcore-nrf)
+# SD-Core NRF K8s Operator
+[![CharmHub Badge](https://charmhub.io/sdcore-nrf-k8s/badge.svg)](https://charmhub.io/sdcore-nrf-k8s)
 
-Charmed Operator for the SD-Core Network Repository Function (NRF).
+Charmed K8s Operator for the SD-Core Network Repository Function (NRF).
```

Comment:
```suggestion
Charmed Operator for the SD-Core Network Repository Function (NRF) for K8s.
```
`Charmed K8s Operator` to me, sounds a bit like it was meant only for Charmed K8s. If it's just my opinion, feel free to ignore this ;)

Reply:
If the naming may cause some misunderstanding, we can change it now.

Reply:
Done